Cannot parse: 6:27: APPLICATION_CREDENTIALS = [,]

I have an action that has been running fine for some time, but yesterday it started giving this issue. My code has not changed between it working and not working;

```
Run docker run --rm -v /home/runner/work/geo_home/geo_home://github/workspace hassfest
##[debug]/usr/bin/bash --noprofile --norc -e -o pipefail /home/runner/work/_temp/1d395f15-1c34-4f91-a3d5-3733ab146cbd.sh
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/usr/src/homeassistant/script/hassfest/__main__.py", line 242, in <module>
    sys.exit(main())
  File "/usr/src/homeassistant/script/hassfest/__main__.py", line 164, in main
    plugin.validate(integrations, config)
  File "/usr/src/homeassistant/script/hassfest/application_credentials.py", line 29, in validate
    config.cache["application_credentials"] = content = generate_and_validate(
  File "/usr/src/homeassistant/script/hassfest/application_credentials.py", line 21, in generate_and_validate
    return format_python_namespace({"APPLICATION_CREDENTIALS": match_list})
  File "/usr/src/homeassistant/script/hassfest/serializer.py", line 98, in format_python_namespace
    return format_python(code, generator=generator)
  File "/usr/src/homeassistant/script/hassfest/serializer.py", line 73, in format_python
    return black.format_str(content.strip(), mode=Mode())
  File "/usr/local/lib/python3.10/site-packages/black/__init__.py", line 1054, in format_str
    dst_contents = _format_str_once(src_contents, mode=mode)
  File "/usr/local/lib/python3.10/site-packages/black/__init__.py", line 1064, in _format_str_once
    src_node = lib2to3_parse(src_contents.lstrip(), mode.target_versions)
  File "/usr/local/lib/python3.10/site-packages/black/parsing.py", line 126, in lib2to3_parse
    raise exc from None
black.parsing.InvalidInput: Cannot parse: 6:27: APPLICATION_CREDENTIALS = [,]
Validating application_credentials...
Error: Process completed with exit code 1.

```

You can see the full details at https://github.com/mmillmor/geo_home/actions/runs/3588747925

The validation job is defined as;

```
name: Validate with hassfest

on:
  push:
  pull_request:
  schedule:
    - cron: "0 0 * * *"

jobs:
  validate:
    runs-on: "ubuntu-latest"
    steps:
      - uses: "actions/checkout@v2"
      - uses: home-assistant/actions/hassfest@master
```

--- a/script/hassfest/serializer.py
+++ b/script/hassfest/serializer.py
@@ -23,7 +23,7 @@ def _wrap_items(
         items = sorted(items)
 
     joined_items = ", ".join(items)
-    return f"{opener}{joined_items}{',' if len(joined_items) != 0 else ''}{closer}"
+    return f"{opener}{joined_items}{',' if joined_items else ''}{closer}"
 
 
 def _mapping_to_str(data: Mapping[Any, Any]) -> str:

--- a/script/hassfest/serializer.py
+++ b/script/hassfest/serializer.py
@@ -21,7 +21,9 @@ def _wrap_items(
     # on one line and some on multiple.
     if sort:
         items = sorted(items)
-    return f"{opener}{','.join(items)},{closer}"
+
+    joined_items = ", ".join(items)
+    return f"{opener}{joined_items}{',' if len(joined_items) != 0 else ''}{closer}"
 
 
 def _mapping_to_str(data: Mapping[Any, Any]) -> str:

